PIM/Operation/04_ManageView.md

```diff
@@ -0,0 +1,179 @@
+  Only those products that are assigned to the selected category or sub-category are displayed in the list of products on the right side.
+
+  ![Select category](../../Assets/Screenshots/PIM/Products/List/SelectCategory.png "[Select category]")
+
+  > [Info] When you create a product, it will be automatically assigned to the currently selected category.
+
+
+
+## Select a variant view
+
+Switch the variant view to either display only the master products and single products in the product list or all products including their variants.
+
+#### Prerequisites
+
+No prerequisites to fulfill.
+
+#### Procedure
+*PIM > Products > Tab LIST*
+
+![Product list](../../Assets/Screenshots/PIM/Products/List/Products.png "[Product list]")
+
+1. Click the *Variants* drop-down list at the top of the product list.   
+  The options below are displayed:
+  - **All products**: All products and their variants are displayed in the product list.   
+  - **Hide variants**: No variants are displayed in the product list.
+
+
+2. Select the desired variant view in the *Variants* drop-down list.
+  Depending on the selection, the product list displays all products or only the master products.
+
+
+
+## Create a view
+
+Create a view with a predefined selection of the language, the channel, the product category, the variant view and the order of columns in the product list.
+
+#### Prerequisites
+
+No prerequisites to fulfill.
+
+> [Info] Depending on the view you want to create, it is necessary to fulfill the corresponding prerequisites in the chapters [Select a language](#select-a-language), [Select a channel](#select-a-channel) or [Select a product category](#select-a-product-category).
+
+#### Procedure
+*PIM > Products > Tab LIST*
+
+![Product list](../../Assets/Screenshots/PIM/Products/List/Products.png "[Product list]")
+
+1. Select the desired settings for the view, you want to save:
+  + Click the *Language* drop-down list and select the desired language.
+  + Click the *Channel* drop-down list and select the desired channel.
+  + Click the *Catalog* drop-down list and select the desired catalog and category.
+  + Click the *Variants* drop-down list and select the desired variant view.   
+
+
+2. If desired, adjust the displayed columns in the product list by following the steps below:
+  + Click the [ ![Columns](../../Assets/Icons/Columns.png "[Columns]") Columns (x)] button.   
+  + Click the *Column* drop-down list, select a column in the list and click the [ADD COLUMN] button to add the column to the columns displayed in the product list.
+  + Click the ![Remove](../../Assets/Icons/Cross03.png "[Remove]") (Remove) button next to a column name to remove the corresponding column from the product list.
+  + Click and hold a column name and by using drag and drop, move the selected column to the desired position. The number left to the column name displays the position number of the column in the product list.
```

Comment:
```suggestion
  + Click and hold a column name and, by using drag and drop, move the selected column to the desired position. The number left to the column name displays the position number of the column in the product list.
```
Alternativ:  + Click and hold a column name and move the selected column to the desired position (by) using drag and drop. The number left to the column name displays the position number of the column in the product list.